### PR TITLE
Fix Junit for stock delete

### PR DESCRIPTION
**Pull request by wwweert123**

### CODE REVIEW

**Review by ryan1604 — APPROVED** [review]
LGTM

File: src/test/java/seedu/financialplanner/investments/WatchListTest.java [hunk 1]
```diff
@@ -1,16 +1,21 @@
...
 import java.util.ArrayList;
 
 import static org.junit.jupiter.api.Assertions.assertNotNull;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
 class WatchListTest {
 
     @Test
+    @Order(1)
```
Comment (ryan1604) [inline src/test/java/seedu/financialplanner/investments/WatchListTest.java:18]: I like the ordering of test methods

File: src/test/java/seedu/financialplanner/investments/WatchListTest.java [hunk 2]
```diff
@@ -20,16 +25,18 @@ void fetchFMPStockPrices() throws FinancialPlannerException {
...
     }
 
     @Test
+    @Order(3)
     void deleteStock() throws FinancialPlannerException {
         WatchList wl = WatchList.getInstance();
-        String stockCode = "AAPL";
-        assertEquals("Apple Inc", wl.deleteStock(stockCode));
+        String stockCode = "GME";
+        assertEquals("Gamestop Corporation - Class A", wl.deleteStock(stockCode));
```
Comment (ryan1604) [inline src/test/java/seedu/financialplanner/investments/WatchListTest.java:40]: Thanks for the quick bug fix!:D

File: src/test/java/seedu/financialplanner/investments/WatchListTest.java [hunk 3]
```diff
@@ -1,16 +1,21 @@
 package seedu.financialplanner.investments;
 
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Order;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
 import seedu.financialplanner.exceptions.FinancialPlannerException;
```
Comment (NeoMinWei) [inline src/test/java/seedu/financialplanner/investments/WatchListTest.java:8]: why is there a space here

File: src/test/java/seedu/financialplanner/investments/WatchListTest.java [hunk 4]
```diff
@@ -20,16 +25,18 @@ void fetchFMPStockPrices() throws FinancialPlannerException {
...
     @Test
+    @Order(2)
     void addStock() throws Exception {
         WatchList wl = WatchList.getInstance();
         String stockCode = "GME";
         assertEquals("Gamestop Corporation - Class A", wl.addStock(stockCode));
     }
 
     @Test
+    @Order(3)
```
Comment (NeoMinWei) [inline src/test/java/seedu/financialplanner/investments/WatchListTest.java:36]: good use of ordering

**Review by NeoMinWei — APPROVED** [review]
LGTM

File: src/test/java/seedu/financialplanner/investments/WatchListTest.java (hunk 3 above)
Comment (wwweert123) [reply src/test/java/seedu/financialplanner/investments/WatchListTest.java:8]: where got space stop farming comments

**Pull request merged**